site/content/3.10/arangograph/data-loader/import.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Start the import
+menuTitle: Start import
+weight: 15
+description: >-
+  Once the data files are provided and the graph is designed, you can start the import
+archetype: default
+---
+
+Before starting the actual import, make sure that:
+- You have selected a database for import or created a new one;
+- You have provided a name for your graph;
+- You have created at least one node;
+- You have created at least one edge;
+- Every file is related to at least one node or edge.
```

Comment:
FYI, the full list of requirements for import is:

- You have selected a database for import or created a new one;
- You have provided a valid name for your graph;
- The graph name provided is not in use already;
- You have created at least one node;
- You have created at least one edge;
- You have uploaded at leat one file;
- Every file is related to at least one node or edge;
- Every node and edge is linked to a file;
- Every node and edge has a valid label;
- Every node and edge label is unique;
- Every node has a primary identifier selected;
- Every edge has an origin and destination file header selected;

Maybe is not needed to include all restrictions to the official list, but this is the list.

Reply:
Thanks, updated the list.